genutils.c

```diff
@@ -62,8 +62,14 @@
 #include "parseutils.h"
 #include "srfsigs.h"
 
+#if PG_VERSION_NUM < 160000
+/* These two functions are used via custom find_option() function
+    Once custom find_option is dropped in PG16+, remove these
+    unused functions here and full definitions below
+    https://github.com/CrunchyData/pgnodemx/pull/18 */
```

Comment:
Yeah I didn't put the note on the find_option function itself, but figured it might be useful to speed up code review when removing later if it's explicitly noted these aren't used anywhere else.